src/CharterManager.sol

```diff
@@ -278,13 +278,15 @@ contract CharterManagerImp {
         _validate(ilk, u, urp, dink, dart, rate, spot, _gate);
     }
 
-    function flux(bytes32 ilk, address src, address dst, uint256 wad) external allowed(src) {
+    function flux(address gemJoin, address src, address dst, uint256 wad) external allowed(src) {
         address surp = getOrCreateProxy(src);
         address durp = getOrCreateProxy(dst);
 
-        VatLike(vat).flux(ilk, surp, durp, wad);
+        VatLike(vat).flux(ManagedGemJoinLike(gemJoin).ilk(), surp, durp, wad);
     }
 
+    function flee(address gemJoin) external {}
```

Comment:
Don't think we need an empty flee here. It's very specific to the crop join.

Reply:
You are right, but I think these will allow us to use a shared cdp manager layer if we ever want to.
We similaraly also have `onLiquidation` and `onVatFlux` that support the shared clipper.